lib/menubar/init.lua

```diff
@@ -132,6 +132,7 @@ local function load_count_table()
                 count_table[name] = count
             end
         end
+        count_file:close()
     end
```

Comment:
Yeah. What I meant (and did not write was): It would be simpler if my idea worked, but you are right that it does not and thus `file.lines(foo)` cannot be used here.